### PR TITLE
Feature/tra 307 add button to profile form

### PR DESCRIPTION
**Pull request by Wojciech-Florczak**
Add `Go to skills matrix` button on profile page

https://apptension.atlassian.net/browse/TRA-307

image

### CODE REVIEW

File: src/shared/components/matrix/myProfileForm/myProfileForm.component.tsx [hunk 1]
```diff
@@ -90,7 +97,18 @@ export const MyProfileForm = () => {
...
+      <ButtonsContainer>
+        <FormActions withoutBackButton nextLabel="Save" isDisabled={!isDirty} isLoading={isSubmitting} />
+        <Button
+          size={ButtonSize.LARGE}
+          variant={ButtonVariant.PRIMARY}
+          type="button"
+          disabled={isSubmitting}
+          onClick={() => history.push(ROUTES.matrix)}
+        >
+          Go to skills matrix <ArrowRightIcon width={22} />
```
Comment (Balldoro) [inline src/shared/components/matrix/myProfileForm/myProfileForm.component.tsx:109]: Button has a property `icon` where you can pass `ButtonIcon.ARROW`, so there's no need in adding icon here

File: src/shared/components/matrix/myProfileForm/myProfileForm.styles.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+export const ButtonsContainer = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-end;
+  flex-wrap: wrap;
+  gap: 40px;
+`;
+
+export const ArrowRightIcon = styled(ArrowRight)`
```
Comment (Balldoro) [inline src/shared/components/matrix/myProfileForm/myProfileForm.styles.ts:12]: and then you can also remove this from here

File: src/shared/components/matrix/myProfileForm/myProfileForm.component.tsx (hunk 1 above)
Comment (Wojciech-Florczak) [reply src/shared/components/matrix/myProfileForm/myProfileForm.component.tsx]: Thanks, I didn't notice that. Changed ✅

File: src/shared/components/matrix/myProfileForm/myProfileForm.styles.ts (hunk 2 above)
Comment (Wojciech-Florczak) [reply src/shared/components/matrix/myProfileForm/myProfileForm.styles.ts]: ✅

**Review by Balldoro — APPROVED** [review]

**Pull request merged**